proxy/src/auth/backend/console.rs

```diff
@@ -177,20 +177,19 @@ impl<'a> Api<'a> {
 
     /// Wake up the compute node and return the corresponding connection info.
     pub(super) async fn wake_compute(&self) -> Result<ComputeConnCfg, WakeComputeError> {
+        let request_id = uuid::Uuid::new_v4().to_string();
         let req = self
             .endpoint
             .get("proxy_wake_compute")
-            .header("X-Request-ID", uuid::Uuid::new_v4().to_string())
+            .header("X-Request-ID", &request_id)
             .query(&[("session_id", self.extra.session_id)])
             .query(&[
                 ("application_name", self.extra.application_name),
                 ("project", Some(self.creds.project().expect("impossible"))),
             ])
             .build()?;
 
-        // TODO: use a proper logger
-        println!("cplane request: {}", req.url());
-
+        info!(id = request_id, url = req.url().as_str(), "request");
         let resp = self.endpoint.execute(req).await?;
         if !resp.status().is_success() {
             return Err(TransportError::HttpStatus(resp.status()).into());
```

Comment:
I wonder, is this error logged somewhere? I could not quickly find it in the proxy code. I see info record above, but that'd be nice to see error too

Reply:
Anton asked the same question essentially. Please see the example above. Every single error like this is propagated to task's root and ends up in `log_error`.